tooling/sync/generate_command_surfaces.py

```
@@ -0,0 +1,148 @@
+                agent=agent,
+                manifest_path=manifest_path,
+                core=core,
+                conditional=conditional,
+                include_conditional=args.enable_conditional,
+            )
+            output_path = output_root / f"{agent}.commands.json"
+            output_path.write_text(
+                json.dumps(payload, indent=2, ensure_ascii=True) + "\n", encoding="utf-8"
+            )
+            print(output_path)
+        return 0
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)